admin/installer/cloudformation.py

```diff
@@ -0,0 +1,293 @@
+"""
+Troposphere script to generate an AWS CloudFormation JSON template.
+
+Sample usage:
+python cloudformation.py > flocker-cluster.cloudformation.json
+
+Resulting JSON template has the following blueprint to describe the
+desired stack's resources and properties:
+1 Control Node with Flocker Control Service, (TLS-enabled) Swarm Manager,
+                    (TLS-enabled) Docker, Ubuntu 14.04
+2 Agent Nodes with Flocker Dataset Agent, Swarm Agent, (TLS-enabled) Docker,
+                   Ubuntu 14.04
+1 Client Node with Flockerctl, Docker, Docker-compose, Ubuntu 14.04
+
+To manifest the blueprint, please input the JSON template at AWS CloudFormation
+Create Stack console (after replacing ``us-east-1`` with your Region):
+https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new
+
+"""
```

Comment:
Also, Add a tip for debugging:

- Turn off rollback on failure.
- See the log (stdout) from the  user data script by going to "get system log" in the AWS console. (or getting the serial output from the command line).